components/network/openssh/patches/0032-revert-dscp.patch

```diff
@@ -0,0 +1,85 @@
+From cbbada124c90a23b78fd4c83f4c4ab9b1a2add23 Mon Sep 17 00:00:00 2001
```

Comment:
Ooops... You don't patch OpenSSH, your patch creates patch to patch openssh.

Reply:
Thanks for spotting that, now it applies as it should.